tests/dummy/app/templates/components-examples/flexberry-simpleolv/configurate-rows.hbs

```diff
@@ -0,0 +1,36 @@
+<h3 class="ui header">{{t "forms.components-examples.flexberry-simpleolv.configurate-rows.caption"}}</h3>
+<form class="ui form flexberry-vertical-form" role="form">
+  <div class="field">
+    <button {{action 'changeAttribute2'}} class="ui button">Change attribute 2</button>
+  </div>
+  <div class="field">
+    {{#settings-example
+      controllerProperties=this
+      componentSettingsMetadata=componentSettingsMetadata
+      componentTemplateText=componentTemplateText
+    }}
+      {{flexberry-simpleolv
+        content=model
+        modelName="ember-flexberry-dummy-suggestion"
+        editFormRoute='ember-flexberry-dummy-suggestion-edit'
+        modelProjection=projection
+        filterByAnyMatch=(action "filterByAnyMatch")
+        filterText=filter
+        sorting=computedSorting
+        sortByColumn=(action "sortByColumn")
+        addColumnToSorting=(action "addColumnToSorting")
+        pages=pages
+        perPageValue=perPageValue
+        perPageValues=perPageValues
+        recordsTotalCount=recordsTotalCount
+        hasPreviousPage=hasPreviousPage
+        hasNextPage=hasNextPage
+        previousPage=(action "previousPage")
+        gotoPage=(action "gotoPage")
+        nextPage=(action "nextPage")
+        configurateRow=(action "configurateRow")
+        componentName="FOLVConfigurateRowsObjectListView"
```

Comment:
I suppose we need to rename all 'FOLV<something>' names to 'SOLV<something>' name to aviod duplication of names with `flexberry-objectlistview` components in usersettings and events sending.